src/main/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributor.java

```diff
@@ -123,34 +113,17 @@ private void processActions(List<Action> actions, InstructionCount requestCount,
   }
 
   /**
-   * Get the oldest page of submitted actions by type - but do not retrieve the
-   * same cases as other CaseSvc' in the cluster
+   * Get the oldest page of submitted actions by type
    *
    * @param actionType the type
    * @return list of actions
-   * @throws LockingException LockingException thrown
    */
-  private List<Action> retrieveActions(ActionType actionType) throws LockingException {
-    Pageable pageable = new PageRequest(0, appConfig.getActionDistribution().getRetrievalMax(), new Sort(
-        new Sort.Order(Direction.ASC, "updatedDateTime")));
-
-    List<BigInteger> excludedActionIds = actionDistributionListManager.findList(actionType.getName(), false);
-    if (!excludedActionIds.isEmpty()) {
-      log.debug("Excluding actions {}", excludedActionIds);
-    }
-    // DO NOT REMOVE THIS NEXT LINE
-    excludedActionIds.add(BigInteger.valueOf(IMPOSSIBLE_ACTION_ID));
-
-    List<Action> actions = actionRepo
-        .findByActionTypeNameAndStateInAndActionPKNotIn(actionType.getName(),
-            Arrays.asList(ActionState.SUBMITTED, ActionState.CANCEL_SUBMITTED), excludedActionIds, pageable);
-    if (!CollectionUtils.isEmpty(actions)) {
+  private List<Action> retrieveActions(ActionType actionType) {
+    List<Action> actions = actionRepo.findSubmittedOrCancelledByActionTypeName(actionType.getName(),
+            appConfig.getActionDistribution().getRetrievalMax());
+    if (CollectionUtils.isNotEmpty(actions)) {
       log.debug("RETRIEVED action ids {}", actions.stream().map(a -> a.getActionPK().toString())
```

Comment:
Does this need to be in the empty check, won't it just print "RETRIEVED action ids" if there are no actions.